toolz/functoolz.py

```diff
@@ -368,12 +368,13 @@ class Compose(object):
     See Also:
         compose
     """
-    __slots__ = 'first', 'funcs'
+    __slots__ = 'first', 'funcs', '__custom_doc'
 
     def __init__(self, funcs):
         funcs = tuple(reversed(funcs))
         self.first = funcs[0]
         self.funcs = funcs[1:]
+        self.__custom_doc = None
```

Comment:
It doesn't work as a class attribute (which is what you mean here i think?) with __slots__; mangling it in __init__ seems to be necessary.